packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,60 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import Logger from 'ember-metal/logger';
+
+var originalLog, logCalls;
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor() {
+    super();
+
+    originalLog = Logger.log;
```

Comment:
Will fix. As soon as I looked back over this commit I knew I should have done that.